aristanetworks/ansible-avd
# Fix(eos_cli_config_gen): Fix the templates for event-handlers

## Change Summary

Current documentation template for `event-handlers` throws error if `poll_interval` is not defined for `trigger_on_counters` and `trigger_on_logging`

```
TASK [arista.avd.eos_cli_config_gen : Generate device documentation] ***************************************************
fatal: [tp-avd-leaf1 -> localhost]: FAILED! => {"msg": "'trigger_config' is undefined. 'trigger_config' is undefined"}
fatal: [tp-avd-spine1 -> localhost]: FAILED! => {"msg": "'trigger_config' is undefined. 'trigger_config' is undefined"}
```

Few updates in config template and its test-cases.

## Related Issue(s)

Fixes #<ISSUE ID>

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes

Changing the logic to add multiple entries in the table for event-handler triggers

## How to test

Do not provide `poll_interval` for `trigger_on_counters` and `trigger_on_logging` input and give other options.
Run the playbook for eos_cli_config_gen, you should not see any error and the documentation output should be proper.

## Checklist

### Repository Checklist
